.github/workflows/handler-test.yml

```diff
@@ -566,7 +566,7 @@ jobs:
           echo "::set-output name=token::$( \
             curl --fail --retry 5 --verbose \
             --insecure \
-            --connect-timeout 5 \
+            --connect-timeout 10 \
```

Comment:
We should increase the other 2 values of `--connect-timeout` to be `10` as well since they also appear after the SOCKS5 proxy is established. 

Note for posterity: this change did not affect the test runs as it was not yet merged to the default branch.